packages/office-ui-fabric-react/src/components/Stack/examples/Stack.Horizontal.Configure.Example.tsx

```diff
@@ -118,23 +148,8 @@ const HorizontalStackConfigureExampleContent: React.FunctionComponent<IExampleOp
   );
 };
 
-function _range(start: number, end: number): number[] {
-  const result = [];
-  for (let i = start; i <= end; i++) {
-    result.push(i);
-  }
-  return result;
-}
-
-// Tokens definition
-const sectionStackTokens: IStackTokens = { childrenGap: 10 };
-const configureStackTokens: IStackTokens = { childrenGap: 20 };
-
-const shadowItemCheckboxStyles: Partial<ICheckboxStyles> = { root: { marginRight: 10 } };
-const wrapItemCheckboxStyles: Partial<ICheckboxStyles> = { root: { marginBottom: 10 } };
-
-export class HorizontalStackConfigureExample extends React.Component<{}, IExampleOptions> {
-  public state: IExampleOptions = {
+export const HorizontalStackConfigureExample: React.FunctionComponent = () => {
+  const [state, setState] = React.useState<IExampleOptions>({
```

Comment:
It seems kind of un-idiomatic for function components to have a single large state object like this. You should probably consider breaking it up into individual `useState` calls (or other patterns if more appropriate for certain members).